safe_transaction_service/account_abstraction/serializers.py

```diff
@@ -32,20 +32,26 @@
 # ================================================ #
 class SafeOperationSignatureValidatorMixin:
     """
-    Mixin class to validate SafeOperation signatures. _get_owners must be overridden to define
+    Mixin class to validate SafeOperation signatures. `_get_owners` can be overridden to define
     the valid owners to sign
     """
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.ethereum_client = EthereumClientProvider()
+        self._deployment_owners: List[ChecksumAddress] = []
 
     def _get_owners(self, safe_address: ChecksumAddress) -> List[ChecksumAddress]:
         """
         :param safe_address:
-        :return: Valid owners to sign the SafeOperation
+        :return:  `init_code` decoded owners if Safe is not deployed or current blockchain owners if Safe is deployed
         """
-        raise NotImplementedError("Must be implemented in subclass")
+        try:
+            return get_safe_owners(safe_address)
```

Comment:
I'm thinking that a better approach would be check if there is `init_code`, as we validate [here](https://github.com/safe-global/safe-transaction-service/blob/229d4bf6a533e1819cb84ccf2c5d26c7a67c5f20/safe_transaction_service/account_abstraction/serializers.py#L128) When `init_code` is missing means not deployed address.
At this way we will reduce the number of calls to the RPC node.

Reply:
That's not true. You can send `init_code` with the address deployed, that's why I did it this way